src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -43,7 +46,32 @@ type ParcelsForDeliveryResponse =
           error: { type: ParcelsForDeliveryErrorType; logId: string };
       };
 
-type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient";
+type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient" | "eventFetchFailed";
+
+type ParcelEventResponse =
+    | {
+          data: Schema["events"][];
+          error: null;
+      }
+    | {
+          data: null;
+          error: { type: ParcelsForDeliveryErrorType; logId: string };
+      };
+
+// Gets all "shipping labels downloaded" events for the given parcels
```

Comment:
can you remove this comment and rename the function `getParcelEvents` to be more descriptive, so that we don't need the comment?